collectors/ioping.plugin/README.md

```diff
@@ -1,86 +1,120 @@
 <!--
 title: "ioping.plugin"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/ioping.plugin/README.md
+description: "Netdata's ioping plugin allows you to measure your disk latency."
 -->
 
 # ioping.plugin
 
-The ioping plugin supports monitoring latency for any number of directories/files/devices,
-by pinging them with `ioping`.
+The ioping plugin supports monitoring latency for any number of directories/files/devices, by pinging them with `ioping`.
+When your disks don't respond in the time you defined in `ioping.conf`, the plugin raises an alert (`ioping_disk_latency`).
 
-A recent version of `ioping` is required (one that supports option `-N`).
-The supplied plugin can install it, by running:
+The collector is enabled if Netdata detects that you have `ioping` installed. Otherwise, it is disabled. 
 
-```sh
-/usr/libexec/netdata/plugins.d/ioping.plugin install
-```
+## Prerequisites
+
+To use this plugin, you need to install `ioping`.
```

Comment:
```suggestion
To use this plugin, you need to install [ioping](https://github.com/koct9i/ioping#ioping).
```

Took libery to make awesome more awesome 😄

Reply:
Thanks, that's awesome!